src/components/BugsTable/bug.graphql

```diff
@@ -0,0 +1,21 @@
+query BugSearch($search: BugSearch!, $paging: Paging) {
```

Comment:
Where is this query being used?

Reply:
sorry, this is in fact for bugzilla, will remove it!